home.css

```diff
@@ -0,0 +1,727 @@
+}
+
+.background-img2 {
+  height: 950px;
+  height: 622px;
+  background: hsl(222, 11%, 18%);
+  margin-top: -5px;
+}
+
+.card {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  width: 20%;
+  height: 82.06px;
+  background: #3b3e45;
+  margin-left: 15px;
+  margin-top: 5px;
+}
+
+.card-img-top {
+  width: 57.08px;
+  height: 51.73px;
+  margin-left: 5px;
+}
+
+.background-img4 {
+  width: 100%;
+  height: 450px;
+  height: 242px;
+  left: 0;
+  background: #414246;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.partner-logos {
+  display: flex;
+  align-items: center;
+  margin-left: 10px;
+  flex-direction: row;
+  justify-content: space-around;
+  height: 80px;
+  width: 100%;
+  margin-top: 80px;
+  position: absolute;
+}
+
+.partner-logo {
+  width: 60px;
+  height: 60px;
+  opacity: 70%;
+  border-radius: 20%;
+}
+
+.partner {
+  width: 76px;
+  height: 34px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #fff;
+}
+
+.partners {
+  width: 100%;
+  height: 5px;
+  justify-content: space-around;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: -80px;
+  margin-bottom: 40px;
+}
+
+.bottom-line3 {
+  width: 40px;
+  height: 2px;
+}
+
+.background-img5 {
+  width: 100%;
+  height: 150px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  background-color: white;
+}
+
+/* Bootstrap */
+.main-sports {
+  width: 95%;
+  height: 90px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  letter-spacing: -0.04em;
+  margin-left: 13px;
+  color: #fff;
+  margin-top: -630px;
+}
+
+.sports-name {
+  width: 70px;
+  height: 41.92px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14.2712px;
+  line-height: 25px;
+  margin-left: 25px;
+  display: flex;
+  align-items: center;
+  letter-spacing: -0.01em;
+  color: #ff5438;
+}
+
+.button1 {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .button2 {
+    cursor: pointer;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 68px;
+    margin-top: 25px;
+    margin-left: 20px;
+    border: none;
+    color: white;
+    background: none;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  .top {
+    background-color: #3e3e3e;
+    flex-direction: row;
+    justify-content: flex-end;
+    gap: 10px;
+    font-family: 'Lato', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    padding-right: 30px;
+    display: flex;
+    align-items: center;
```

Comment:
- [ ] It looks like the mobile version of your website doesn't fit well with the screens. 😢 
![image](https://user-images.githubusercontent.com/90222110/205447052-3a1668e0-df2d-43db-b812-68398a828a92.png)

Please fix your `media queries` in order for it to look well on the mobile screens 😃 👍